pages/index.js

```diff
@@ -1,7 +1,40 @@
-import React from 'react';
+import React, { useCallback, useRef } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getMovies } from '../store/actions/MovieActions';
+import { makeSelectMovieList } from '../store/selectors/MovieSelector';
+import { debounce } from '../utils/debounce';
 
 const Index = () => {
-  return <div>Home Page</div>;
+  const dispatch = useDispatch();
+  const movies = useSelector(makeSelectMovieList);
+  const inputRef = useRef();
+
+  const handleFetch = useCallback(() =>
+    dispatch(getMovies({ limit: 3, page: 0, title: inputRef.current.value }))
```

Comment:
page i ajde moze ovde ali limit bi trebalo da drzis u samom servisu jer se to retko menja ili u sagi nesto tipa perPage

Reply:
tj reduceru